etabotsite/etabotsite/settings.py

```diff
@@ -220,6 +220,28 @@
 ]
 
 
+# OAuth
+AUTHLIB_OAUTH_CLIENTS = {}
+JIRA_OAUTH = custom_settings.get('JIRA_APP_OAUTH')
```

Comment:
please update the `custom_settings` in our slack channel as well

Reply:
will do. I'll need to figure out prod vs dev auth settings.

Reply:
created ticket not to forget https://etabot.atlassian.net/browse/ET-577